flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendStopWithSavepointTest.java

```diff
@@ -133,6 +134,31 @@ public void testStopWithExplicitSavepointType() throws Exception {
                     assertThat(jobID, is(jid));
                     assertThat(advanceToEndOfEventTime, is(false));
                     assertThat(savepointDirectory, is("test-target-dir"));
+                    assertThat(formatType, is(expectedFormat));
+                    stopWithSavepointLatch.trigger();
+                    return CompletableFuture.completedFuture(savepointDirectory);
+                });
+        MockedCliFrontend testFrontend = new MockedCliFrontend(clusterClient);
+        testFrontend.stop(parameters);
+        stopWithSavepointLatch.await();
+    }
+
+    @Test
+    public void testStopWithExplicitSavepointTypeLongOption() throws Exception {
```

Comment:
This is also true for these two test mthods, we can certainly avoid code duplication via introducing a base private test method with a parameter of option key.